x-pack/plugins/security_solution/server/lib/product_features_service/product_features_service.ts

```diff
@@ -135,14 +140,24 @@ export class ProductFeaturesService {
     // The `securitySolutionProductFeature:` prefix is used for ProductFeature based control.
     // Should be used only by routes that do not need RBAC, only direct productFeature control.
     const APP_FEATURE_TAG_PREFIX = 'securitySolutionProductFeature:';
-    // The "access:securitySolution-" prefix is used for API action based control.
-    // Should be used by routes that need RBAC, extending the `access:` role privilege check from the security plugin.
-    // An additional check is performed to ensure the privilege has been registered by the productFeature service,
-    // preventing full access (`*`) roles, such as superuser, from bypassing productFeature controls.
+
+    /** @deprecated Use security.authz.requiredPrivileges instead */
     const API_ACTION_TAG_PREFIX = `access:${APP_ID}-`;
 
+    const isAuthzEnabled = (authz?: RecursiveReadonly<RouteAuthz>): authz is AuthzEnabled => {
+      return Boolean((authz as AuthzEnabled)?.requiredPrivileges);
```

Comment:
Would like to know what's the difference between `security.authz.requiredFeatures` and `security.authz.requiredPrivileges`?

Reply:
Sorry, the PR title was wrong, the correct entry is `security.authz.requiredPrivileges`, `security.authz.requiredFeatures` does not exist